lib/lita/handlers/espn_fantasy_football.rb

```diff
@@ -148,32 +146,38 @@ def espn_scoreboard_scrape(week)
           ],
           "rows" => []
         }
+
         params = {
           "leagueId" => config.league_id,
           "seasonId" => config.season_id
         }
+
         # If no period (aka, week) is specified, ESPN defaults to the
         # most recent (aka, current) matchup period
         unless week.empty?
           params["matchupPeriodId"] = week
         end
 
-        param_string = params.map { |key, val| "#{key}=#{val}" }
+        param_string = params.map { |key, val| "#{key}=#{val}" }.join("&")
         url = "http://games.espn.go.com/ffl/scoreboard?#{param_string}"
-        scoreboard = Nokogiri::HTML(open(url))
+        Lita.logger.debug("Searching for score at #{ url }")
+
+        page = Nokogiri::HTML(open(url))
+        matchups = page.xpath('//*[@class="ptsBased matchup"]')
```

Comment:
If by faster, you mean easier for me to figure out with chrome's developer tools, then yes.